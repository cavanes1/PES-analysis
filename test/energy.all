-285.285937020161
-285.285936810663
-285.285936760043
-285.285936760061
-285.285936826256
-285.285936826397
-285.285936839845
-285.285936363456
-285.285936848091
-285.285936846974
-285.285936848102
-285.285936846905
-285.285936830628
-285.285936839024
-285.285936955349
-285.285936955058
-285.285936970603
-285.285936870657
-285.285936970636
-285.285936870529
-285.285936971875
-285.285936868267
-285.285936971808
-285.285936868061
-285.285936960140
-285.285936909196
-285.285936971086
-285.285936959459
-285.285936213116
-285.285936516337
-285.285936516351
-285.285936619358
-285.285936619487
-285.285936632546
-285.285936160347
-285.285936633571
-285.285936637366
-285.285936633582
-285.285936637296
-285.285936563749
-285.285936629583
-285.285936745880
-285.285936745594
-285.285936748553
-285.285936661255
-285.285936748588
-285.285936661123
-285.285936763496
-285.285936658732
-285.285936763431
-285.285936658525
-285.285936750641
-285.285936699713
-285.285936761587
-285.285936749960
-285.285936069545
-285.285936504046
-285.285936535439
-285.285936561826
-285.285936586002
-285.285936094823
-285.285936582797
-285.285936582239
-285.285936587624
-285.285936590495
-285.285936617289
-285.285936548664
-285.285936695244
-285.285936694971
-285.285936711693
-285.285936610513
-285.285936723890
-285.285936610460
-285.285936699970
-285.285936608271
-285.285936711859
-285.285936607970
-285.285936698610
-285.285936649054
-285.285936711554
-285.285936699348
-285.285936069577
-285.285936561719
-285.285936535584
-285.285936586018
-285.285936094841
-285.285936587633
-285.285936589714
-285.285936582827
-285.285936582188
-285.285936617307
-285.285936609254
-285.285936695266
-285.285936694978
-285.285936723872
-285.285936610606
-285.285936711741
-285.285936610401
-285.285936711939
-285.285936608189
-285.285936699918
-285.285936608081
-285.285936701436
-285.285936649072
-285.285936710399
-285.285936699365
-285.285936241536
-285.285936607402
-285.285936623634
-285.285936179564
-285.285936654086
-285.285936648772
-285.285936657837
-285.285936652680
-285.285936620594
-285.285936697619
-285.285936761459
-285.285936762045
-285.285936776392
-285.285936676766
-285.285936775395
-285.285936676593
-285.285936791997
-285.285936674396
-285.285936775888
-285.285936674121
-285.285936783663
-285.285936715341
-285.285936775645
-285.285936765553
-285.285936241802
-285.285936623779
-285.285936179707
-285.285936657103
-285.285936652892
-285.285936654238
-285.285936648845
-285.285936620731
-285.285936592862
-285.285936761597
-285.285936761321
-285.285936775506
-285.285936676863
-285.285936776569
-285.285936676778
-285.285936776092
-285.285936674470
-285.285936792074
-285.285936674332
-285.285936748996
-285.285936715517
-285.285936778849
-285.285936765692
-285.285936271145
-285.285936156171
-285.285936667166
-285.285936666644
-285.285936667178
-285.285936666575
-285.285936690915
-285.285936658712
-285.285936775030
-285.285936774736
-285.285936791474
-285.285936690353
-285.285936791506
-285.285936690225
-285.285936789189
-285.285936687956
-285.285936789124
-285.285936687747
-285.285936779842
-285.285936728983
-285.285936790802
-285.285936779189
-285.285934627187
-285.285936190950
-285.285936189104
-285.285936190964
-285.285936189039
-285.285936178226
-285.285936182323
-285.285936298646
-285.285936298354
-285.285936314361
-285.285936213945
-285.285936314395
-285.285936213814
-285.285936315975
-285.285936211558
-285.285936315908
-285.285936211352
-285.285936303442
-285.285936252501
-285.285936314429
-285.285936302798
-285.285936321118
-285.285936673977
-285.285936675073
-285.285936674408
-285.285936665173
-285.285936687026
-285.285936783279
-285.285936782986
-285.285936798541
-285.285936698585
-285.285936798879
-285.285936698588
-285.285936799906
-285.285936696199
-285.285936798748
-285.285936695997
-285.285936788983
-285.285936737132
-285.285936798788
-285.285936787391
-285.285936315314
-285.285936674487
-285.285936673328
-285.285936641409
-285.285936653625
-285.285936782164
-285.285936781870
-285.285936796409
-285.285936697473
-285.285936797329
-285.285936697339
-285.285936799566
-285.285936695209
-285.285936799507
-285.285936694877
-285.285936786363
-285.285936736010
-285.285936797372
-285.285936786274
-285.285936321144
-285.285936673920
-285.285936665180
-285.285936646913
-285.285936783290
-285.285936782999
-285.285936798856
-285.285936698728
-285.285936798584
-285.285936698470
-285.285936798829
-285.285936696214
-285.285936799856
-285.285936696003
-285.285936787172
-285.285936738010
-285.285936799258
-285.285936787404
-285.285936315180
-285.285936642202
-285.285936677993
-285.285936782094
-285.285936781803
-285.285936797224
-285.285936697398
-285.285936796371
-285.285936697275
-285.285936799504
-285.285936695018
-285.285936799430
-285.285936694939
-285.285936787477
-285.285936735943
-285.285936798358
-285.285936786205
-285.285936270627
-285.285936649499
-285.285936765810
-285.285936765528
-285.285936765283
-285.285936681092
-285.285936765317
-285.285936680959
-285.285936792549
-285.285936678821
-285.285936792484
-285.285936678614
-285.285936770570
-285.285936719579
-285.285936781550
-285.285936769926
-285.285936295399
-285.285936774225
-285.285936773946
-285.285936784583
-285.285936689636
-285.285936794384
-285.285936689281
-285.285936778206
-285.285936687211
-285.285936803217
-285.285936686849
-285.285936760391
-285.285936728063
-285.285936796601
-285.285936778322
-285.285936761748
-285.285936890255
-285.285936905788
-285.285936708216
-285.285936905824
-285.285936708091
-285.285936907064
-285.285936866316
-285.285936906996
-285.285936866111
-285.285936895332
-285.285936844387
-285.285936906276
-285.285936894649
-285.285936759147
-285.285936905498
-285.285936768658
-285.285936905526
-285.285936842319
-285.285936906770
-285.285936719633
-285.285936906707
-285.285936887354
-285.285936895054
-285.285936758376
-285.285936905981
-285.285936882807
-285.285936820819
-285.285936821155
-285.285936923768
-285.285936820966
-285.285936919393
-285.285936818717
-285.285936919537
-285.285936818500
-285.285936913447
-285.285936859637
-285.285936920681
-285.285936909899
-285.285936422628
-285.285936821128
-285.285936687292
-285.285936823240
-285.285936755101
-285.285936822300
-285.285936769786
-285.285936810644
-285.285936728694
-285.285936821578
-285.285936802313
-285.285936820886
-285.285936821061
-285.285936920501
-285.285936818744
-285.285936919361
-285.285936818547
-285.285936907753
-285.285936859665
-285.285936922406
-285.285936909935
-285.285936422374
-285.285936822243
-285.285936769865
-285.285936822184
-285.285936752974
-285.285936810506
-285.285936790565
-285.285936821457
-285.285936817472
-285.285936825160
-285.285936820022
-285.285936920741
-285.285936819768
-285.285936915372
-285.285936860913
-285.285936923833
-285.285936911173
-285.285936412005
-285.285936819908
-285.285936758657
-285.285936808240
-285.285936712331
-285.285936819190
-285.285936805670
-285.285936824161
-285.285936819749
-285.285936908275
-285.285936860850
-285.285936921703
-285.285936911108
-285.285936411592
-285.285936808057
-285.285936802071
-285.285936818992
-285.285936809260
-285.285936779956
-285.285936849217
-285.285936927813
-285.285936899425
-285.285936576367
-285.285936860125
-285.285936846629
-285.285936823857
-285.285936910486
-285.285936777210
