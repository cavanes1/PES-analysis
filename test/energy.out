  -0.100000E+00  -0.189021E+04   0.362704E+04
  -0.900000E-01  -0.192519E+04   0.379825E+04
  -0.800000E-01  -0.195587E+04   0.397225E+04
  -0.700000E-01  -0.198226E+04   0.414905E+04
  -0.600000E-01  -0.200434E+04   0.432865E+04
  -0.500000E-01  -0.202212E+04   0.451104E+04
  -0.400000E-01  -0.203560E+04   0.469624E+04
  -0.300000E-01  -0.204479E+04   0.488423E+04
  -0.200000E-01  -0.204967E+04   0.507503E+04
  -0.100000E-01  -0.205026E+04   0.526862E+04
   0.000000E+00  -0.204654E+04   0.546501E+04
   0.100000E-01  -0.203853E+04   0.566420E+04
   0.200000E-01  -0.202621E+04   0.586619E+04
   0.300000E-01  -0.200960E+04   0.607097E+04
   0.400000E-01  -0.198868E+04   0.627856E+04
   0.500000E-01  -0.196347E+04   0.648894E+04
   0.600000E-01  -0.193396E+04   0.670212E+04
   0.700000E-01  -0.190014E+04   0.691810E+04
   0.800000E-01  -0.186203E+04   0.713688E+04
   0.900000E-01  -0.181962E+04   0.735846E+04
   0.100000E+00  -0.177291E+04   0.758284E+04
